pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -134,7 +135,7 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 			glog.V(4).Infof("The attributes of queue <%s> in proportion: deserved <%v>, allocate <%v>, request <%v>, share <%0.2f>",
 				attr.name, attr.deserved, attr.allocated, attr.request, attr.share)
 
-			deserved.Add(attr.deserved)
+			deserved.Add(queueDeserved)
```

Comment:
If `!attr.deserved.LessEqual(attr.request)`, `queueDeserved` need be changed to `attr.deserved` increased value.

Reply:
Sorry, I did not get it. Could you explain a bit more?

I think the `queueDeserved` is already the increased value. In the for loop we add it to `attr.deserved`  and `deserved`. After the for loop, we calculate the `remaining` by subtracting  `deserved`

Reply:
Suppose that `queueDeserved` is `{cpu: 2, mem: 2G}`, `attr.deserved` is `{cpu: 1, mem: 1G}` and `attr.request` is `{cpu: 2, mem: 2G}`, `attr.deserved` will be `{cpu: 3, mem: 3G}` after `attr.deserved.Add(queueDeserved)`(line 128). Then `attr.deserved` will be `{cpu: 2, mem: 2G}` after `attr.deserved = helpers.Min(attr.deserved, attr.request)`(line 130). So `attr.deserved`'s increased value is actually `{cpu: 1, mem: 1G}`, and we just need add `{cpu: 1, mem: 1G}` instead of `{cpu: 2, mem: 2G}` to `deserved`.

Reply:
got it, thanks.  Let me update it.

Reply:
Updated.

Reply:
Thanks! You code works. Just give some suggestion.